packages/components/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,395 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconArrowDown, IconArrowUp } from '@frontify/fondue-icons';
+import { type ReactNode, forwardRef, useMemo, type KeyboardEvent, type CSSProperties, useRef } from 'react';
+
+import { useSyncRefs } from '#/hooks/useSyncRefs';
+import { useTextTruncation } from '#/hooks/useTextTruncation';
+
+import styles from './styles/table.module.scss';
+import { handleKeyDown } from './utils';
+
+type TableRootProps = {
+    /**
+     * Whether the table should have a fixed or auto layout
+     * @default 'auto'
+     */
+    layout?: 'auto' | 'fixed';
+    /**
+     * Whether header should stick to the top when scrolling
+     */
+    sticky?: 'head' | 'col' | 'both';
+    /**
+     * Optional caption text for the table that appears above it
+     */
+    caption?: string;
+    /**
+     * Whether to make the table take full width of its container
+     * @default true
+     */
+    fullWidth?: boolean;
+    children: ReactNode;
+    'aria-label'?: string;
+    'aria-describedby'?: string;
```

Comment:
what abot using `.. & ({ 'aria-label': string } | { 'aria-labelledby': string });` like in the other components so at leas one needs to be added